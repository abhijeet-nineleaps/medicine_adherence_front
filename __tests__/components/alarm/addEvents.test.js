import Enzyme, {shallow} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import React from 'react';
import configureStore from 'redux-mock-store';
import toJson from 'enzyme-to-json';
import Addevent from '../../../src/components/alarm/addEvents';
Enzyme.configure({adapter: new Adapter()});
const mockStore = configureStore([]);
describe('test collector category', () => {
  it('test category', () => {
    const wrapper = shallow(<Addevent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});