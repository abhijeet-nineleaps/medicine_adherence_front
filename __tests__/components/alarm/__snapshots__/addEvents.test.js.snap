// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test collector category test category 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
